Makefile

```diff
@@ -232,3 +232,32 @@ jenkins-output.txt:
 
 reports/%.csv: registry/ontologies.ttl sparql/%.sparql
 	arq --data $< --query sparql/$*.sparql --results csv > $@.tmp && mv $@.tmp $@
+
+################################################
+#### Commands for building the Docker image ####
+################################################
+VERSION = "0.0.1"
```

Comment:
not very descriptive. Suggetion: `OBOFOUNDRY_IMAGE_VERSION`

Reply:
do not review draft pull requests :P

Reply:
oops I didn't look at the flag

Reply:
its like looking at someone six pack in progress under their t-shirt on January 10.